packages/vanilla/src/components/global-nav/top-nav/notifications/_list/_list.scss

```diff
@@ -1,3 +1,25 @@
+@import '~basics/colors/colors.scss';
+
 .hig__notifications__list {}
 
-.hig__notifications__loading {}
\ No newline at end of file
+.hig__notifications__list-content {
+  &:empty {
+    min-height: 240px;
+
+    &:before {
+      content: url("./bell.svg");
+      display: flex;
+      justify-content: center;
+      padding: 4em 0 2em 0;
```

Comment:
WIP padding; there's a "Notifications" title that is fixed in the flyout that we haven't included yet.

Reply:
I think we are using pixels instead of ems?